manifests/plugin/dns_cloudflare.pp

```diff
@@ -21,6 +22,7 @@
   Optional[String[1]] $email        = undef,
   Stdlib::Absolutepath $config_path = "${letsencrypt::config_dir}/dns-cloudflare.ini",
   Boolean $manage_package           = true,
+  String $package_ensure            = $letsencrypt::package_ensure,
   Integer $propagation_seconds      = 10,
 ) {
   require letsencrypt::install
```

Comment:
This file require `letsencrypt::install` and not `letsencrypt`, maybe something to double-check?